configs/pgd/metafile.yml

```diff
@@ -17,6 +17,8 @@ Collections:
 
 Models:
   - Name: pgd_r101-caffe_fpn_head-gn_4xb3-4x_kitti-mono3d
+    Alias:
+       - pgd-kitti
```

Comment:
Suggest using `pgd_kitti` to keep the distinction about `_` and `-`. In LiDAR-based detection, the `alias` is a little complex and such modification is helpful. For example, `pointpillars_waymod5-3d-3class`, `pointpillars_waymod5-3d-car`